.github/workflows/build-and-test.yml

```diff
@@ -110,14 +110,11 @@ jobs:
           subscription-id: f557c728-871d-408c-a28b-eb6b2141a087
 
       # Run tests
-      - name: Test Library and Library with coverage
-        run: ./gradlew --build-cache library:test --scan
-
-      - name: Test Harness Tests with coverage
-        run: ./gradlew --build-cache testharness:test --scan
+      - name: Run Library and Test Harness tests with coverage
+        run: ./scripts/build test
```

Comment:
Note that we currently aren't running service unittests, @sergiygetlin .

Reply:
Execution of tests moved to https://github.com/DataBiosphere/terra-landing-zone-service/pull/421/files#diff-cc1bc65976b3ec6248bb0079f2f78eeab12f1f9da665739637b0cb576f816c06R54

Reply:
@sergiygetlin I have now included running the service unit tests.

<img width="934" alt="image" src="https://github.com/DataBiosphere/terra-landing-zone-service/assets/484484/31567249-1b74-4565-a1bf-0e4ae81ebd9f">